### ISSUE TITLE
[Request] Linking barrels 1.10.2

### ISSUE THREAD

**Issue by craftboy1012** [jaquadro/StorageDrawers#574]
Hey,
I'm playing with your mod and I'm having troubble with storage because my farm is far away of my home. It would be cool if there was a way to link 2 barrels so they share the same amount items from a long distance.

Amazing mod
Keep this beauty running. Your awesome

craftboy

**Comment by jaquadro** [comment]
Heh, can't quite get away from the JABBA associations.

I think there's several issues open now suggesting this feature.  It's a pretty popular request.  I even started laying the groundwork for it a while ago, but it's been on the back-burner.

**Comment by craftboy1012** [comment]
Yeah the JABBA mod is always associated with yours, wich is kinda normal. Hope you can do that functionality in the future.

You could try to explore some of the JABBA mod's code (even if it's in 1.7.10)
